### DISCUSSION TITLE
V1 对整个表单进行异步校验

### DISCUSSION THREAD

**Discussion by Mikawawawa** [alibaba/formily#1955] [Q&A 1.x] [answered]
我现在有一个场景，我想在表单校验、表单提交的时候触发一个异步的校验逻辑，并且在部分字段上创建校验异常，这种情况下，我应该怎么操作比较好呀，现在没有找到方法实现

**Comment by janryWang** [comment, marked as answer]
x-rules支持写异步校验的，提交会自动触发
